prime-router/docs/onboarding-users/receivers.md

```diff
@@ -1,12 +1,307 @@
+* The new organization must have at least one `receiver` defined, but the `receiver` does not need to have a transport 
+defined - the download site can be used until an automated delivery mechanism is set up.
+* There are two fields that will be used as "keys" to further work, below.   The `-name` of the organization 
+(eg, `lt-phd`) will be used as the Okta `group`, and the `translation.schemaName:` value (eg, `lt/lt-covid-19`) will be 
+used as the schema name in the next step.
+* Below is an example of the organization file
+  
+```yaml
+- name: lt-pdh
+  description: LT Department of Health
+  jurisdiction: STATE
+  stateCode: LT
+  receivers:
+    - name: elr
+      topic: covid-19
+      jurisdictionalFilter: [ "matches(ordering_facility_state, LT)" ]
+      translation:
+        type: HL7
+        useBatchHeaders: true
+        receivingApplicationName: LT-PDH
+        receivingApplicationOID:
+        receivingFacilityName: LT-PDH
+        receivingFacilityOID:
+```
+* In the above example, the jurisdictional filter searches the `ordering_facility_state` field in the report for anything 
+that matches the code LT.
+* Filters can be applied to the organization or receiver. For more information on filters see: 
+(https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/playbooks/how-to-use-filters.md)
+* In addition, there is the translation section, which specifies the output format that will be sent to the receiver. 
+Currently, we have three formats available:
+    - HL7
+    - CSV
+    - FHIR
+* The quality filters verify that test results have the minimum fields required by most public health jurisdictions. 
+* Those fields are made adjustable to be customized per jurisdictional specifications.
+* These filters are applied by default to the receiver and set in code.
+* They do not need to be added to the receiver setting.
+* If a custom rule is added to the quality filter for a receiver, all default quality filters will be ignored. The default filter includes 
+these requirements:
+- The following fields all have a value:
+     - Patient First Name
+     - Patient Last Name
+     - Patient DOB
+     - Test Result
+     - Specimen Type
+     - Specimen ID/Accession Number
+     - Device Identifier
+    
+- At least one of the following fields has a value:
+     - Patient Street Address
+     - Patient Zip Code
+     - Patient Phone
+     - Patient Email
+    
+- At least one of the following date fields has a valid date:
+     - Test Ordered Date
+     - Specimen Collection Date
+     - Test Result Date
+    
+- The following fields are 10 characters in length:
+     - Testing Lab CLIA
+     - Ordering Facility CLIA
+    
+- Processing Code does not equal T nor D
+
+- Only test results that pass all the above requirements will be transferred to the jurisdiction. There are two options 
+for jurisdictions that want all results reported.
+- Use AllowAll() to bypass all quality filters 
```

Comment:
We don't support AllowAll()  in the UP.

Reply:
Is there only 1 option for jurisdiction that want all results reported? That lone option is the secondary feed with the reverseQualityFilter() set to true?